File: BitBoy/app/src/main/java/lib/core/PageObject.java
package lib.core;


import android.graphics.Rect;

import java.util.Map;




public class PageObject{
	public int pageID;
	public Map<String,Object> info;
	public int dr;
	public Boolean isHistory;
	public Boolean isHome;

	public Boolean isGestureUsed = true;
	
    public PageObject(int _pageID) {
    	pageID=_pageID;
    	dr=2;
    	isHistory=true;
    	isHome=false;
    	
    }

    
}

File: SMSManager/app/src/main/java/lib/core/PageObject.kt
package lib.core

class PageObject(var pageID: Int)
{
    var info: Map<String, Any>? = null
    var dr: Int = 2
    var isHistory: Boolean = true
    var isHome: Boolean = false
    var isGestureUsed: Boolean = true

    init
    {

    }

}